### ISSUE TITLE
Review the current prefixes

### ISSUE THREAD

**Issue by napoleaowingert** [pluginsandsnippets/widget-for-zendesk-chat-via-api#26]
We should review somewhen the current prefixes that are in use and maybe replace them in case they are too long.

=> We should not use weak / too short prefixes (eg. 3 letters is pretty short these days and that's why we are using eg. instead of 'eas', we are using 'edd_eas', so 6 letters and in the edd context in this case).
=> We should also avoid too long prefixes (mostly because option names and meta keys will likely be really long since we also have the name/suffix that identify what they are - this part is already long sometimes)
=> We should use one single prefix for option names and meta keys etc so its easy to identify things
=> Same for css classes/ids etc, php classes, functions,  and JS functions/variables in the global context. If needed, we can use slightly different prefixes per context, eg. 1 single prefix for php, 1 single prefix for JS, 1 single prefix for CSS.
eg. sometimes we need to write it in lowercase only, etc. but prefereably they all should be the same.

Bottom line is that we must always use the same prefix everywhere if possible, finding a balance in the prefix size between avoiding conflicts (not too short) and still keep it user/developer friendly (not too long)

**Comment by napoleaowingert** [comment]
I think we should wait and see if this plugin will have more features in the future or not.

At this moment it looks like it has a small market. So we should put this issue on hold.
